include/core/HY_Features.hpp

```diff
@@ -59,24 +60,14 @@ namespace hy_features {
         /**
          * @brief Construct a new HY_Features object from a Network and a set of formulations.
          * 
-         * Constructs the HY_Catchment objects for each catchment feature in the network, and attaches tha formaulation
+         * Constructs the HY_Catchment objects for each catchment connecting them with the provided link_key attaches tha formaulation
```

Comment:
The grammar is the revised comment doesn't make sense anymore.

Also,
```suggestion
         * Constructs the HY_Catchment objects for each catchment connecting them with the provided link_key attaches the formulation
```